### ISSUE TITLE
License not detected when part of an array

### ISSUE THREAD

**Issue by rogierschouten** [davglass/license-checker#82]
license-checker is not picking up the license from this package: 

https://github.com/faeldt/base64id/blob/master/package.json

```json
{
    "name": "base64id"
  , "version": "0.1.0"
  , "licenses" : [
        {
            "type" : "MIT",
            "url" : "https://github.com/caolan/async/raw/master/LICENSE"
        }
    ]
  , "description": "Generates a base64 id"
  , "author": "Kristian Faeldt <faeldt_kristian@cyberagent.co.jp>"
  , "repository": {
        "type": "git"
      , "url": "https://github.com/faeldt/base64id.git"
   }
  , "main": "./lib/base64id.js"
  , "engines": { "node": ">= 0.4.0" }
}
```

**Comment by rogierschouten** [comment]
I see this is deprecated syntax in the JSON file. Perhaps it is wise to still support it in the license-checker module to be able to check older modules?

**Comment by davglass** [comment]
`license-checker` does actually support that syntax :)
https://github.com/davglass/license-checker/blob/master/lib/index.js#L111-L119

The issue is that the `package.json` in that repo doesn't match what's in the registry:
http://registry.npmjs.org/base64id/latest

There isn't a `licenses` key in the package.json from the registry. If you install it like this:

`npm i base64id` then run `license-checker` you get:

```
├─ base64id@0.1.0
│  ├─ licenses: UNKNOWN
│  ├─ repository: https://github.com/faeldt/base64id
│  ├─ publisher: Kristian Faeldt
│  └─ email: faeldt_kristian@cyberagent.co.jp
```

But if you install it from the repo with: `npm i faeldt/base64id` and run `license-checker` you get:

```
├─ base64id@0.1.0
│  ├─ licenses: MIT
│  ├─ repository: https://github.com/faeldt/base64id
│  ├─ publisher: Kristian Faeldt
│  ├─ email: faeldt_kristian@cyberagent.co.jp
│  └─ licenseFile: /Users/davglass/src/tmp/node_modules/base64id/LICENSE
```
